python/lsst/ctrl/mpexec/cli/cmd/commands.py

```diff
@@ -124,8 +126,33 @@ def build(ctx: click.Context, **kwargs: Any) -> None:
     _unhandledShow(show, "build")
 
 
+def start_coverage(coverage_packages: Tuple) -> Any:
+    print("Coverage turned ON!", file=sys.stderr)
```

Comment:
Maybe issue a single message combining the coverage package list. Also please remove the `!`. So an output along the lines of:

```
Coverage enabled of packages ...
```
or
```
Coverage enabled.
```

Since this is in the click layer I think we might want to use `click.echo()` rather than `print()` ?